.circleci/config.yml

```diff
@@ -793,31 +789,54 @@ workflows:
           requires:
             - playwright-build
 
-  build-and-deploy-all-apps-workflow:
```

Comment:
Old workflow diagram looks similar to this screenshot. `app-deploy` are separate workflows triggered by `build-and-deploy-all-apps-workflow` workflow. 

<img width="1112" alt="Screenshot 2023-02-23 at 5 48 20 PM" src="https://user-images.githubusercontent.com/35533885/221048245-59bb1258-a567-4e80-937c-c771271867f4.png">